core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -480,10 +480,15 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         | Spark standalone and Mesos only:
         |  --total-executor-cores NUM  Total cores for all executors.
         |
+        | Spark standalone and YARN only:
+        |  --executor-cores NUM        Number of cores to use on each executor. Default:
+        |                              1 in YARN mode; in standalone mode, all cores in a worker
+        |                              allocated to an application will be assigned to a single
+        |                              executor.
```

Comment:
I would rephrase this as:

Number of cores per executor. (Default: 1 in YARN mode, or all available cores on the worker in standalone mode)